### PR TITLE
Test if binfmts has been set

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

**Comment by sylvestre** [comment]
Fails because it uses lli-9 from the system:

`/usr/bin/lli-9: ./foo.bc: error: Unknown attribute kind (68) (Producer: 'LLVM13.0.0' Reader: 'LLVM 9.0.0')`

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -26,7 +26,8 @@ jobs:
             sudo apt-add-repository "deb http://apt.llvm.org/${{ env.UBUNTU_CODENAME }}/ llvm-toolchain-${{ env.UBUNTU_CODENAME }}-${{ matrix.version }} main"
           fi
           sudo apt-get update
-          sudo apt-get install -y clang-${{ matrix.version }} clang-tidy-${{ matrix.version }} clang-format-${{ matrix.version }} clang-tools-${{ matrix.version }} llvm-${{ matrix.version }}-dev lld-${{ matrix.version }} lldb-${{ matrix.version }} llvm-${{ matrix.version }}-tools libomp-${{ matrix.version }}-dev libc++-${{ matrix.version }}-dev libc++abi-${{ matrix.version }}-dev libclang-common-${{ matrix.version }}-dev libclang-${{ matrix.version }}-dev libclang-cpp${{ matrix.version }}-dev
+          sudo apt --purge -y remove "llvm-*-runtime"
```
Comment (serge-sans-paille) [inline .github/workflows/CI.yml:29]: Could you add a comment to that line?

**Comment by sylvestre** [comment]
@serge-sans-paille your change broke the CI ;)

File: .github/workflows/CI.yml (hunk 1 above)
Comment (sylvestre) [reply .github/workflows/CI.yml:29]:
```suggestion
          # Remove the previous installed llvm runtime to make sure that the installed
          # runtime sets the right version of binfmts
          sudo apt --purge -y remove "llvm-*-runtime"
```

**Comment by serge-sans-paille** [comment]
Just checked: `binfmts` support for LLVM is not active on RHEL, nor on Fedora. So what we should do is a `REQUIRES: debian` or something like that

**Comment by sylvestre** [comment]
@serge-sans-paille we have support for distro detection ? :)

**Comment by serge-sans-paille** [comment]
No, we should implement that I guess

**Comment by serge-sans-paille** [comment]
Instead of a platform check, we should check if `/proc/sys/fs/binfmt_misc/register` has a reference to llvm, and if so run the test, otherwise don't run it.

**Pull request closed without merging**